ert_storage/endpoints/priors.py

```diff
@@ -0,0 +1,53 @@
+from uuid import UUID
+from typing import Mapping, Type
+from fastapi import APIRouter, Depends
+from ert_storage.database import Session, get_db
+from ert_storage import database_schema as ds, json_schema as js
+from ert_storage.json_schema.prior import (
+    ConstPrior,
+    TriangularPrior,
+    NormalPrior,
+    LognormalPrior,
+    TruncatedNormalPrior,
+    UniformPrior,
+    DUniformPrior,
+    ErrfPrior,
+    DErrfPrior,
+)
+
+router = APIRouter(tags=["prior"])
+
+
+@router.get("/experiments/{experiment_id}/priors")
+def get_priors(
+    *, db: Session = Depends(get_db), experiment_id: UUID
+) -> Mapping[str, dict]:
+    experiment = db.query(ds.Experiment).filter_by(id=experiment_id).one()
+    return experiment_priors_to_dict(experiment)
+
+
+PRIOR_FUNCTION_TO_PYDANTIC: Mapping[ds.PriorFunction, Type[js.Prior]] = {
+    ds.PriorFunction.const: ConstPrior,
+    ds.PriorFunction.triangular: TriangularPrior,
+    ds.PriorFunction.normal: NormalPrior,
+    ds.PriorFunction.lognormal: LognormalPrior,
+    ds.PriorFunction.truncated_normal: TruncatedNormalPrior,
+    ds.PriorFunction.uniform: UniformPrior,
+    ds.PriorFunction.duniform: DUniformPrior,
+    ds.PriorFunction.errf: ErrfPrior,
+    ds.PriorFunction.derrf: DErrfPrior,
```

Comment:
As a total noob to these terms, i have no clue what prefix `D` tells me, and neither `Errf`. Is it possible / does it make sense giving them more meaningful names? If it is normal to use these - nevermind my comment and i will learn 👍

Reply:
d = discrete

Reply:
I agree with Øystein in that it's not obvious. I copied the names verbatim and later on guessed that `D` stands for discrete. I'm not sure how we should elaborate this to the user, since prefixing the types with `Discrete` might be a bit wordy. (Or maybe not?)

Reply:
I guess if it is normal to prefix discrete value-names with `D` it make sense only having it. But - if it is a ERT ™️  term, i guess every one of the developers touching this code in the future will wonder what it means and guess ( and / or ) ask someone what it means. I don't like that if it can easily be avoided 👍